service/controller/resource/prometheus/resource.go

```diff
@@ -75,6 +75,13 @@ func toPrometheus(v interface{}) (*promv1.Prometheus, error) {
 					KeyFile:  fmt.Sprintf("/etc/prometheus/secrets/%s/key", key.Secret()),
 				},
 			},
+			Replicas: &replicas,
+			Resources: corev1.ResourceRequirements{
+				Requests: corev1.ResourceList{
+					corev1.ResourceCPU:    resource.MustParse("1000m"),
+					corev1.ResourceMemory: resource.MustParse("1Gi"),
```

Comment:
We will be limited quite fast by this.
e.g. archon has 7g memory, so we can only monitor 7 clusters

Reply:
the entire cluster has 7g of memory?

Reply:
I remember the idea here. As we only support CAPI clusters, we were reckoning we'd be able to transition slowly. It's not that we'd suddenly double up on prometheus

Reply:
What with customers who create fancy 3 node clusters?

Reply:
I feel like we might actually need to be smart about this. Any 'default' value is either gonna be too small for big clusters, or too big for small clusters.

Reply:
Closing here for now